helpdesk_mgmt_ticket_invoice/views/helpdesk_ticket_view.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="helpdesk_ticket_action_form" model="ir.actions.act_window">
+        <field name="name">Helpdesk Ticket</field>
+        <field name="res_model">helpdesk.ticket</field>
+        <field name="view_mode">form</field>
+    </record>
+
+    <record id="helpdesk_ticket_form_view_inherit" model="ir.ui.view">
+        <field name="name">helpdesk.tickeT.form.view</field>
+        <field name="model">helpdesk.ticket</field>
+        <field name="inherit_id" ref="helpdesk_mgmt.ticket_view_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='tag_ids']" position="after">
+                <field name="type" invisible="1"/>
+                <field name="invoice_id"
+                        attrs="{'invisible': [('type', 'not in', ['out_invoice','in_refund'])]}"
```

Comment:
This is wrong, we don't want hide invoice selection depending on invoice type... we want being able to select invoices only when they belong to certain type. I was talking about a domain restriction (when `Many2one` combobox is displayed, only invoices whose type is `out_invoice` or `in_refund` should be only displayed). And I'll add another condition: invoices should be already posted (`state=posted`).

If you don't understand, please ask